observability/opentelemetry-api/pom.xml

```diff
@@ -33,48 +33,63 @@
         <version>28.0.0.Beta1-SNAPSHOT</version>
     </parent>
 
-    <artifactId>wildfly-opentelemetry-api-legacy-namespace</artifactId>
+    <artifactId>wildfly-opentelemetry-api</artifactId>
 
     <name>WildFly: OpenTelemetry API</name>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>${project.groupId}</groupId>
+                <artifactId>wildfly-standard-ee-bom</artifactId>
+                <version>${ee.maven.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
```

Comment:
The observability/pom.xml has a dependencyManagement import of wildfly-legacy-ee-bom. That can either be deleted, and all the child modules control that, as you do here, or you can change that to wildfly-standard-ee-bom and drop this whole block.

Reply:
I tried removing that, but that breaks the micrometer build, and we trying to avoid changing that module as part of this work, though I guess it's safe/clean enough. I'll do that if you're ok with it.